ui/src/components/Publication/Visualization/index.tsx

```diff
@@ -25,6 +26,10 @@ type BoxProps = BoxEntry & {
 };
 
 const Box: React.FC<BoxProps> = (props): React.ReactElement => {
+    const updateXarrow = useXarrow();
+    useEffect(() => {
+        updateXarrow();
+    }, [updateXarrow]);
```

Comment:
"updateXarrow" dependency is creating an infinite render loop and makes the page freeze
![image](https://github.com/JiscSD/octopus/assets/48569671/7894668d-9504-4ba2-b70f-9c68a90a2baf)

From what I see on "react-xarrows" documentation, we just need to call "useXarrow()" inside this component. We don't need the useEffect since "useXarrow" hook will be automatically called every time a visualization box is mounted into the DOM.
"updateXarrow" example is for other use cases like click or drag events.

Reply:
I see, I wasn't too sure about that one, that makes sense. Just pushed the fix